frontend/src/recipePage.jsx

```diff
@@ -0,0 +1,89 @@
+import './recipePage.css';
+import React, { useState, useEffect } from 'react';
+
+export default function RecipePage({name, desc, image, ingredients, steps}) {
+  const [recipes, setRecipes] = React.useState();
+  const [ingredientList, setIngredientList] = React.useState('');
+  const [anIngredient, setNewIngredient] = React.useState('');
+  const [step, setSteps] = React.useState(' ');
+  const [anInstruction, setNewInstruction] = React.useState('');
+
+
+
+  const addIngredient = async() => {
```

Comment:
The put request works great, however you probably noticed that you have to refresh the page to see the changes made. We can keep the  DOM manipulation code from Milestone 2, as well as these new changes so that the change is reflected on both the frontend and the backend!